Example/Example Swift/Info.plist

```diff
@@ -49,7 +49,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.14.0</string>
+	<string>0.15.0</string>
```

Comment:
Maybe it's about time for the example app to grow up and get a version 1? (Probably for next release)